backend/app/models.py

```diff
@@ -142,3 +151,12 @@ class Image(db.Model):
 
     def __repr__(self):
         return '<Image %r>' % self.name
+
+
+class RevokedTokenModel(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    jwt_id = db.Column(db.String(120))
+
+    @classmethod
+    def is_jti_blacklisted(cls, jwt_id):
```

Comment:
By context (class where this is defined) it is obvious that this method is about JWT, so you could call it `is_blacklisted`